README.md

```diff
@@ -5,7 +5,10 @@ This project requires Node v16.x.x (npm v8.x.x). You can use Node Version manage
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
 nvm install 16
 ```
-...and then you can do `npm install` etc as below.
+...and then you can do `npm install` etc as below. \
```

Comment:
Typo at end of line.

The text of line 10 looks good to me but you don't need to prepend the ellipsis since it's not part of the previous topic.

Reply:
ok 👌

Reply:
You still have that typo, ie:

![image](https://user-images.githubusercontent.com/10002889/199331057-a21fece1-c9ef-4e44-8b52-86227d1f7341.png)

Reply:
You mean the backslash "\"? I wanted to make a new line after "below."

Reply:
I removed the backslash. Looks like "double enter" can already create a new line.

Reply:
You can view the file in GH to see it rendered, or add VS Code extension "Markdown Preview Enhanced" to see it.